src/SupabaseStorageClient.ts

```diff
@@ -1,7 +1,7 @@
 import { StorageBucketApi, StorageFileApi } from './lib'
 import { Fetch } from './lib/fetch'
 
-export class SupabaseStorageClient extends StorageBucketApi {
+class SupabaseStorageClient extends StorageBucketApi {
```

Comment:
Since `StorageClient` would now be the canonical name, could this be `export class StorageClient`?

Reply:
Fair enough. I chose to do it like this because:

1. If we're making an alias anyway, there is no point in changing things more than they need to be changed. As in, adding the `SupabaseStorageClient as StorageClient` would mean adding a single line and changing nothing as opposed to renaming the base class to `StorageClient` and adding the `StorageClient as SupabaseStorageClient`. Just my thought process behind this.

2. I personally like having all the exports in a single place as opposed to having an `export class ...` and then `exports {StorageClient as SupabaseStorageClient}`.

These are all personal preference, though.

Reply:
[Fixed](https://github.com/supabase/storage-js/pull/48/commits/0a13fd3782763dba232668d90271816e9b73126e)